### ISSUE TITLE
Add all aria attributes

### ISSUE THREAD

**Issue by Zettexe** [Kotlin/kotlinx.html#261]
Seems all the aria attributes such as `aria-orientation` and `aria-labelledby` are missing. These should be added so we dont have to declare them as `attributes["aria-orientation"]`.
